src/theme.tsx

```diff
@@ -31,6 +31,20 @@ const theme = {
       xl: "h-12 w-12",
     },
   },
+  badge: {
+    base: "font-sans font-medium text-xs rounded-full",
+    variants: {
+      primary: "text-white bg-gray-800",
+      secondary: "text-white bg-blue-500",
```

Comment:
```suggestion
      secondary: "text-gray-700 bg-gray-50",
```

Something that looks like secondary variant 😄